cpp/CMakeLists.txt

```
@@ -407,7 +475,7 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/include/raft/version_config.hpp
 
 include("${rapids-cmake-dir}/export/write_dependencies.cmake")
 
-set(raft_components distance nn)
+set(raft_components distance nn compiled)
 set(raft_install_comp raft raft)
```

```suggestion
set(raft_install_comp raft raft raft)
```

The `raft_components` and `raft_install_comp` need to be equal lengths

---

I expect this is the bug as it would cause the final sets to look like:

```
-- raft-distance-targets.cmake => install component raft
-- raft-nn-targets.cmake => install component raft
-- raft-compiled-targets.cmake => install component distance
-- raft-distance-lib-targets.cmake => install component nn
-- raft-nn-lib-targets.cmake => install component compiled
-- raft-compiled-lib-targets.cmake => install component 
```

---

Duh... such a silly mistake. Sorry about that. Thanks a lot @robertmaynard ! This has fixed the build issue for me. Now somehow the pytest isn't able to find the right modules. I'll figure out that.